package.json

```diff
@@ -74,7 +77,7 @@
   "scripts": {
     "setDependencies": "npm i && npm install -g gulp-cli && npm install -g jasmine",
     "lint": "jshint src --reporter=node_modules/jshint-stylish",
-    "build": "npm run lint && gulp generate-build_version && gulp build",
+    "build": "npm run lint && gulp generate-build_version && gulp build_min",
```

Comment:
You need to build and minify. Like this `"build": "npm run lint && gulp generate-build_version && gulp build && gulp minify"`